CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ CHANGELOG](http://keepachangelog.com/) for how to update this file. This project
 adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
-
+### Added
+- Add Celery integration. (#124)
```

Comment:
Not sure if this is going to create any issues, but can you please add an empty line here? (between lines 8 and 9)